src/Artsy/Router/RouterLink.tsx

```diff
@@ -1,23 +1,43 @@
 import { Link, LinkProps } from "found"
+import PropTypes from "prop-types"
 import React from "react"
 
 /**
- * Wrapper component around found's <Link> component that can be easily styled
- * via styled-components; e.g.,
+ * Wrapper component around found's <Link> component with a fallback to a normal
+ * <a> tag if ouside of a routing context.
+ *
+ * NOTE: If using styled-components, <RouterLink> can be easily styled like so:
  *
  * const StyledLink = styled(RouterLink)`
  *   ...
  * `
  */
-export const RouterLink: React.FC<LinkProps> = ({
-  to,
-  className,
-  children,
-  ...props
-}) => {
-  return (
-    <Link to={to} className={className} {...props}>
-      {children}
-    </Link>
-  )
+
+export const RouterLink: React.FC<LinkProps> = (
+  { to, children, ...props },
+  context
+) => {
+  const isRouterContext = Boolean(context.router)
+
+  if (isRouterContext) {
+    return (
+      <Link to={to} {...props}>
+        {children}
+      </Link>
+    )
+  } else {
+    return (
+      <a href={to as string} {...props}>
```

Comment:
It surprised me that `to` wasn't already a string. It looks like it's [of type `LocationDescriptor`](https://github.com/4Catalyzer/found/blob/master/types/index.d.ts#L345). I don't know that we've ever used it as anything but a string, so we should be fine blindly casting it as a string here....right?

Reply:
That was my thinking here. I think if we ever need anything more advanced -- like passing around `LocationDescriptors` -- we'll be inside of a router context but we'll have to cross that when we get there.